src/core_plugins/metrics/public/lib/html_id_generator.js

```diff
@@ -0,0 +1,12 @@
+import uuid from 'uuid';
```

Comment:
perhaps we could take this out of `metrics/`? It could be useful to resolve https://github.com/elastic/kibana/issues/13474 (duplicate id-s in angular dropdown).

Reply:
I really would like to have a global service for this, that solves this. I hoped, that this could be one of the first things I could add to the new platform approach once it's merged :-)

Until then I am not sure what would be a good place to put this, so we don't need to import weird not-really-apis? I got the feeling, that the TSVB is kind of a self contained plugin not importing from the core until now (@simianhacker correct me if I am wrong), so I didn't want to destroy this.